src/test/java/com/wikia/webdriver/testcases/discussions/ReportingPostTests.java

```diff
@@ -715,21 +713,15 @@ private void moderatorCanDeleteReportedPostOnPostDetailsPage() {
   }
 
   private UserPostsPage openUserPostsAndWaitUntilLoaded(String authorId) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused private "openUserPostsAndWaitUntilLoaded" method. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AUnusedPrivateMethod)